bench-ci/run-assumeutxo-bench.sh

```diff
@@ -53,7 +53,7 @@ setup_assumeutxo_snapshot_run() {
       -DBUILD_BENCH=OFF \
       -DBUILD_TESTS=OFF \
       -DCMAKE_BUILD_TYPE=RelWithDebInfo \
-      -DCMAKE_CXX_FLAGS="-fno-omit-frame-pointer" \
+      -DCMAKE_CXX_FLAGS="-fno-omit-frame-pointer -g3" \
```

Comment:
note that both will likely affect performance (and probably shouldn't be the defaults)